services/src/atdd/src/test/resources/features/mvp.1/FlowReinstall.feature

```diff
@@ -0,0 +1,64 @@
+@FREINSTALL
+Feature: Flow re-reinstalling after switch comes back up.
+
+  @MVP1.1
+  Scenario: Re-installing Ingress and Egress flows.
+
+  This scenario creates simple network and ingress/egress flows. Next this scenario turns off and on back, then checks whether
```

Comment:
Can you describe the topology a bit better in this description?  ie a-b-c .. and we turn off b ( or a or whatever).